reactkarma/reactkarma.py

```diff
@@ -191,6 +191,17 @@ async def _check_reaction(
         emoji = reaction.emoji
         upvote = await self._is_upvote(guild, emoji)
         if upvote is not None:
+            message = "{0}({1})".format(user.display_name, user.name)
+
+            if upvote == added:
+                message += " gave a karma to "
+            else:
+                message += " took a karma from "
+            
+            message += "{0}({1})".format(author.display_name, author.name)
+            message += "."
```

Comment:
You can also improve how you're building the message. Have all the different components in separate variables and then compose the message like you did with `.format()` before you send it out.